packages/app/src/runner/SpecRunner.vue

```diff
@@ -126,44 +145,57 @@ const autStore = useAutStore()
 const screenshotStore = useScreenshotStore()
 const runnerUiStore = useRunnerUiStore()
 const preferences = usePreferences()
+const initialSpecsListWidth: number = props.gql.localSettings.preferences.specListWidth ?? 280
+const initialReporterWidth: number = props.gql.localSettings.preferences.reporterWidth ?? 320
+const reporterWidth = ref(initialReporterWidth)
+const specListWidth = ref(initialSpecsListWidth)
 
+// Todo: maybe `update` should take an object, not just a key-value pair and do updates like this all in one batch
 preferences.update('autoScrollingEnabled', props.gql.localSettings.preferences.autoScrollingEnabled ?? true)
 preferences.update('isSpecsListOpen', props.gql.localSettings.preferences.isSpecsListOpen ?? true)
-
-const runnerPane = ref<HTMLDivElement>()
+preferences.update('reporterWidth', initialReporterWidth)
+preferences.update('specListWidth', initialSpecsListWidth)
 
 const autMargin = 16
+const collapsedNavBarWidth = 64
 
 const containerWidth = computed(() => {
-  // TODO: make these values dynamic in UNIFY-592:
-  const reporterWidth = 320
-  const navWidth = 64
-  const specsListWidth = runnerUiStore.isSpecsListOpen ? 280 : 0
   const miscBorders = 4
-  const nonAutWidth = reporterWidth + navWidth + specsListWidth + (autMargin * 2) + miscBorders
+  const nonAutWidth = reporterWidth.value + specListWidth.value + (autMargin * 2) + miscBorders + collapsedNavBarWidth
 
-  return width.value - nonAutWidth
+  return windowWidth.value - nonAutWidth
 })
 
 const containerHeight = computed(() => {
-  // TODO: make these values dynamic in UNIFY-592:
+  // TODO: in UNIFY-595 the header's contents will be finalized
+  // at narrow widths content will start to wrap
```

Comment:
Should we put all these constants in a single place?

Reply:
The header, when it has final content, is maybe going to wrap at smaller sizes, so it almost needs to be measured from the DOM. We need to allow for the playground UI being open and expanding too, which this doesn't do yet. I agree though we should gather whatever remains that needs to be in a constant into a single place. We can do that now or in the later PR.

Or one other option is to directly set some values in pinia and get them from there. So eg "the nav bar width is 64 in pinia" would tell both the nav bar and the runner what the size is, instead of "the nav bar is in the close state in pinia" being used by multiple components to decide the width must be 64px. And similar for other values that are needed for the runner like this.